shap/explainers/kernel.py

```diff
@@ -117,7 +117,7 @@ def __init__(self, model, data, link=IdentityLink(), **kwargs):
         else:
             model_null = self.model.f(self.data.data)
         if isinstance(model_null, (pd.DataFrame, pd.Series)):
-            model_null = model_null.values
+            model_null = np.squeeze(model_null.values)
```

Comment:
Could you clarify what this change is designed to fix?

Reply:
Sorry for the confusing comments earlier, Scott. Looks like after earlier refactoring of kernel.py, for models with dataframe output, one extra dimension is somehow added, which eventually makes the resulting dimension of shap value different. This change fix the dimension issue and will only impact the specific case with dataframe output.

thanks.

Reply:
makes sense. Thanks!